components/radio/stories/radio.stories.js

```diff
@@ -91,31 +95,132 @@ export default {
+			isEmphasized: false,
+			label: "Default, selected",
 			customStyles: {
 				"max-width": "220px",
-			}
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-2",
+		})}
+		${Template({
+			...args,
+			isChecked: true,
+			isEmphasized: true,
+			label: "Emphasized, selected",
+			customStyles: {
+				"max-width": "220px",
+			},
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-3",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
-			label: "Disabled"
+			label: "Disabled",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-4",
 		})}
 		${Template({
 			...args,
 			isDisabled: true,
 			isReadOnly: true,
-			label: "Read only"
+			label: "Read-only",
+			name: (typeof args.name != "undefined" ? args.name + "-" : "") + "test-5",
 		})}
 	</div>
 `;
+
+const BasicGroupTemplate = (args) => html`
+	<div style="display: flex; flex-direction: column; align-items: flex-start;">
+		${Template({
+			...args,
+			label: "Example label",
+			id: "radio-1-" + args?.id ?? "default",
+			name: "radio-example-" + args?.name ?? "default",
+		})}
+		${Template({
+			...args,
+			isChecked: true,
+			label: "Inititally selected radio button that has wrapping label text",
+			customStyles: {
+				"max-width": "220px",
+			},
+			id: "radio-2-" + args?.id ?? "default",
+			name: "radio-example-" + args?.name ?? "default",
+		})}
+	</div>
+`;
+
+// ===== Stories ======
+
+export const Default = (args) => {
+	return window.isChromatic() ? ChromaticGroupTemplate(args) : BasicGroupTemplate(args);
+};
+Default.tags = ["!autodocs"];
+
+export const Standard = BasicGroupTemplate.bind({});
+Standard.args = {
+	name: "standard",
+};
+Standard.tags = ["!dev"];
+
+export const Sizing = BasicGroupTemplate.bind({});
+Sizing.args = {
+	name: "sizing",
+};
+Sizing.tags = ["!dev"];
+Sizing.decorators = [withSizingWrapper];
+
+/**
+ * The emphasized option provides a visual prominence for the selected radio button. 
+ * It is best for forms, settings, lists or grids of assets, and other situations where a
+ * radio button needs to be noticed.
+ */
+export const Emphasized = BasicGroupTemplate.bind({});
+Emphasized.args = {
+	isEmphasized: true,
+	name: "emphasized",
+};
+Emphasized.tags = ["!dev"];
+Emphasized.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+export const Disabled = BasicGroupTemplate.bind({});
+Disabled.args = {
+	isDisabled: true,
+	name: "disabled",
+};
+Disabled.tags = ["!dev"];
+Disabled.parameters = {
+	chromatic: { disableSnapshot: true },
+};
+
+/**
+ * A radio group has a read-only option for when it's in the disabled state but still needs to be shown.
+ * This allows for content to be copied, but not interacted with or changed.
+ * 
+ * - Read-only radio buttons are disabled, but not all disabled radio buttons are read-only.
+ * - Read-only radio buttons do not have a focus ring, but the button should be focusable.
```

Comment:
I don't understand this line here- the read-only radio buttons don't have a button, right?

<img width="281" alt="Screenshot 2024-07-10 at 11 13 36 AM" src="https://github.com/adobe/spectrum-css/assets/69602589/64f5d4f6-659d-4e9b-8bca-704900a52eab">

Reply:
I've left these migrated comments for now because after trying to harmonize the differences between what's there and what's documented, it revealed a bigger issue with our read-only states (and within SWC). It was unclear what the correct behavior should be. Updating this documentation is a part of CSS-833: _reevaluate different implementations' use of read-only._